### DISCUSSION TITLE
Using private ECR images as a Service in Github Actions

### DISCUSSION THREAD

**Discussion by BeyondEvil** [aws-actions/configure-aws-credentials#554] [General]
If you come here looking for a way to use private ECR images as ["services"](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idservices) in Github Actions workflows, here's a workaround:

https://github.com/community/community/discussions/19819#discussioncomment-3969355

**Comment by peterwoodworth** [comment]
Thanks for sharing this!
